express-zod-api/src/index.ts

```diff
@@ -1,5 +1,3 @@
-import "./zod-plugin";
```

Comment:
moved to DTS banner in TSUP config.

Reply:
check if it's imported unconditionally in the distributable JS code of the framework